src/components/SystemsPage/SystemsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Table, TableBody } from '@material-ui/core';
+
+import { EmptyState, InfoCard } from '@backstage/core-components';
+
+import { useCortexApi } from '../../utils/hooks';
+import { SystemsPageRow } from './SystemsPageRow';
+import { useDetailCardStyles } from '../../styles/styles';
+
+import { GroupByOption } from '../../api/types';
+
+export const SystemsPage = () => {
```

Comment:
so rather than a subpage in `CortexPage`, we'll need something equivalent to `EntityCortexContent` -- a tab for "systems" similar to how we have a tab for "components":
![image](https://user-images.githubusercontent.com/2001609/151936875-3594c64a-bac9-4ddb-8451-b8990e030bc3.png)

and we'll need to export the new component in both `plugin.ts` directly, as well as in `extendableCortexPlugin` (also in `plugin.ts`)

Reply:
so the subpage in CortexPage should be removed? Or should we have both?

Reply:
yup we should remove this, only want the tab